client/src/components/ReviewTile.js

```diff
@@ -0,0 +1,24 @@
+// prepare formatted review tiles 
+import React, { useEffect, useState } from "react";
+
+const ReviewTile = ({ review }) => {
+  let body, username, createdAtString, score
+  
+  if(review) {
```

Comment:
when would there be a case where we try to create a ReviewTile, but there is no `review`? we should remove this `if` statement as well